pkgs/applications/version-management/forgejo/default.nix

```
@@ -57,7 +58,13 @@ buildGoModule rec {
     "-X 'main.Tags=${lib.concatStringsSep " " tags}'"
   ];
 
-  postInstall = ''
+  postInstall = lib.optionalString withPreCompress ''
+     # Create static gzip and brotli files
+     find public -type f -regextype posix-extended -iregex '.*\.(css|html|js|svg|ttf|txt)' | while read file; do
```

(non-blocking): Could be written as for-loop, but it doesn't really matter.
If you prefer your version, ignore this suggestion :)

```suggestion
     for file in $(find public -type f -regextype posix-extended -iregex '.*\.(css|html|js|svg|ttf|txt)'); do
```